tools/istio-iptables/pkg/constants/constants.go

```diff
@@ -78,10 +78,22 @@ const (
 	DryRun                    = "dry-run"
 	Clean                     = "clean"
 	RestoreFormat             = "restore-format"
+	SkipRuleApply             = "skip-rule-apply"
+	SkipValidation            = "skip-validation"
 )
 
 // Constants for iptables commands
 const (
 	IPTABLESRESTORE  = "iptables-restore"
 	IP6TABLESRESTORE = "ip6tables-restore"
 )
+
+// Constants for syscall
+const (
+	// sys/socket.h
```

Comment:
not: this is accessible from go standard library I think

Reply:
It's not defined in `golang.org/x/sys/unix`, where it would be defined.

Reply:
I didn't find `= 0x50` or `= 80` with var name something like `ORIGIN`
There might be other approach of obtain the value. Let me know then